lib/tasks/gettext.rake

```diff
@@ -90,4 +90,25 @@ namespace :gettext do
       end
     end
   end
+
+  task :extract_locale_names do
+    require 'yaml/store'
+    require_relative '../vmdb/fast_gettext_helper.rb'
+
+    Vmdb::FastGettextHelper.register_locales
+
+    locale_hash = {}
+    FastGettext.available_locales.each do |locale|
+      FastGettext.locale = locale
+      # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Português)
+      human_locale = _("locale_name")
+      human_locale = locale if human_locale == "locale_name"
+      locale_hash[locale] = human_locale
+    end
+
+    store = YAML::Store.new("config/human_locale_names.yaml")
```

Comment:
Cool, does this auto-update the YAML?  I can't believe I haven't used this before.  👍